pipelines/build/common/build_base_file.groovy

```diff
@@ -151,7 +161,9 @@ class Builder implements Serializable {
                 ENABLE_TESTS: enableTests,
                 ENABLE_INSTALLERS: enableInstallers,
                 ENABLE_SIGNER: enableSigner,
-                CLEAN_WORKSPACE: cleanWorkspace
+                CLEAN_WORKSPACE: cleanWorkspace,
+                CLEAN_WORKSPACE_AFTER: cleanWsAfter,
+                CLEAN_WORKSPACE_BUILD_OUTPUT_ONLY: cleanWorkspaceBuildOutputAfterBuild
```

Comment:
Hmm it worries me that after the response to @karianna 's comment these two names are now inconsistent. I quite liked the"after" in the name for consistency with other variables but either way we do need to consider how new people looking at the scripts will see this.

Reply:
@sxa do we want CLEAN_WORKSPACE_BUILD_OUTPUT_ONLY_AFTER ?

Reply:
quite a mouthful!

Reply:
BUILD_OUTPUT_ONLY sort of implies at the end of the build

Reply:
i'll change it to: CLEAN_WORKSPACE_BUILD_OUTPUT_ONLY_AFTER

Reply:
changed